src/system.jl

```diff
@@ -319,6 +367,31 @@ Base.@kwdef struct GeneratorStatusRT <: GeneratorStatus
     regulation_commitment::KeyedArray{Bool, 2}
 end
 
+struct LoadServices
+    pmin::KeyedArray{Float64, 2}
+    pmax::KeyedArray{Float64, 2}
+    regulation_up_offers::KeyedArray{Union{Missing, PriceSensitiveBid}, 2}
+    regulation_down_offers::KeyedArray{Union{Missing, PriceSensitiveBid}, 2}
+    spinning_offers::KeyedArray{Union{Missing, PriceSensitiveBid}, 2}
+    on_supplemental_offers::KeyedArray{Union{Missing, PriceSensitiveBid}, 2}
+    off_supplemental_offers::KeyedArray{Union{Missing, PriceSensitiveBid}, 2}
+end
+
+struct EnergyBids
```

Comment:
Is this a good name?  When we refer to "bids" it can mean bids in either direction.  But in the market data "bid" means demand and "offer" means supply.